### ISSUE TITLE
Version 1.3/0 

### ISSUE THREAD

**Issue by 22two** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#383]

**Comment by jtc-m** [comment]
happening here too, any idea for cause?

**Comment by AbdullahAlfaraj** [comment]
@22two , @jtc-m my guess is, it could be because your preview images are in PNG format and the output image is in JPG format.
you can change format like this:

![image](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/7842232/e75cc365-27a8-4dc6-83d9-1cb092bb31ce)

**Comment by jtc-m** [comment]
> @22two , @jtc-m my guess is, it could be because your preview images are in PNG format and the output image is in JPG format. you can change format like this:
> 
> ![image](https://user-images.githubusercontent.com/7842232/271000365-e75cc365-27a8-4dc6-83d9-1cb092bb31ce.png)

Ah yes that seems to be it!

**Comment by 22two** [comment]
> @22two , @jtc-m my guess is, it could be because your preview images are in PNG format and the output image is in JPG format. you can change format like this:
> 
> ![image](https://user-images.githubusercontent.com/7842232/271000365-e75cc365-27a8-4dc6-83d9-1cb092bb31ce.png)

Yes, it works! However, now the PNG size is six times larger than that of JPG. In future updates, can we switch back to using JPG as the file format?

**Comment by jurandfantom** [comment]
thank you for point out about PNG instead jpeg - worth to add information about that in installation instruction

**Comment by chichi233** [comment]
!!!!!!!!!!  yes!!!  thanks !!!!!!!!!!!!!!!

> @22two,@jtc-m 我的猜测是，这可能是因为您的预览图像是 PNG 格式，而输出图像是 JPG 格式。 您可以像这样更改格式：
> 
> ![图像](https://private-user-images.githubusercontent.com/7842232/271000365-e75cc365-27a8-4dc6-83d9-1cb092bb31ce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI2MjY5MjQsIm5iZiI6MTcwMjYyNjYyNCwicGF0aCI6Ii83ODQyMjMyLzI3MTAwMDM2NS1lNzVjYzM2NS0yN2E4LTRkYzYtODNkOS0xY2IwOTJiYjMxY2UucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIxNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMTVUMDc1MDI0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjE3ZWI3ZmVhZjExNGQwMmI4YjliYTdlYjg5NTBlYTMxNzU3M2NkYWMyMTE0M2IyMzYzMWUzYzlmNjdhNDhjMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.lsLq8V8PfKphgTnoFNDH2iEB-zqgS1IArQpDaq7WzCQ)
